recipes/flytekit/meta.yaml

```
@@ -0,0 +1,87 @@
+{% set name = "flytekit" %}
+{% set version = "0.26.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flytekit-{{ version }}.tar.gz
+  sha256: 83bf9ed9d014b0683f49a668113266f4a2f394d4870cd733a9b73b438fe8ef23
+
+build:
+  # Skipping Windows for now
+  # xref: conda-forge/protobuf-feedstock#133
+  skip: true  # [py<38 or win]
+  entry_points:
+    - pyflyte-execute=flytekit.bin.entrypoint:execute_task_cmd
+    - pyflyte-fast-execute=flytekit.bin.entrypoint:fast_execute_task_cmd
+    - pyflyte-map-execute=flytekit.bin.entrypoint:map_execute_task_cmd
+    - pyflyte=flytekit.clis.sdk_in_container.pyflyte:main
+    - flyte-cli=flytekit.clis.flyte_cli.main:_flyte_cli
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - checksumdir >=1.2.0
+    - click >=6.6,<8.0
+    - cloudpickle >=2.0.0
+    - cookiecutter >=1.7.3
+    - croniter >=0.3.20,<4.0.0
+    - dataclasses-json >=0.5.2
+    - deprecated >=1.0,<2.0
+    - diskcache >=5.2.1
+    - docker-image-py >=0.1.10
+    - docstring_parser >=0.9.0
+    - flyteidl >=0.21.4
+    - grpcio >=1.3.0,<2.0
+    - keyring >=18.0.1
+    - marshmallow-jsonschema >=0.12.0
+    - natsort >=7.0.1
+    - pandas >=1.0.0,<2.0.0
+    - protobuf >=3.6.1,<4
+    - pyarrow >=4.0.0,<7.0.0
+    - python
+    - python-dateutil <=2.8.1,>=2.1
+    - python-json-logger >=2.0.0
+    - pytimeparse >=1.1.8,<2.0.0
+    - pytz
+    - requests >=2.18.4,<3.0.0
+    - responses >=0.10.7
+    - retry ==0.9.2
+    - six >=1.9.0,<2.0.0
+    - sortedcontainers >=1.5.9,<3.0.0
+    - statsd >=3.0.0,<4.0.0
+    - urllib3 >=1.22,<2.0.0
+    - wheel >=0.30.0,<1.0.0
+    - wrapt >=1.0.0,<2.0.0
```

```suggestion
    - wrapt >=1.0.0,<2.0.0
    - __unix
    # xref: conda-forge/protobuf-feedstock#133
```

Ref: https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html